docs/scale-cluster.rst

```diff
@@ -24,12 +24,12 @@ may be required: see :ref:`Important things to know <scale-cluster-important>`.
    :local:
 
 
-.. _scale-cluster-settings:
+.. _scale-cluster-preferences:
 
-Find the Cluster Settings page
-==============================
+Find the Cluster Preferences page
+=================================
 
-To scale your clusters in the Console, you must find the Cluster Settings page.
+To scale your clusters in the Console, you must find the Cluster Preferences page.
```

Comment:
perhaps it would read better if we wrote

```suggestion
To scale your clusters in the Console, navigate to the Cluster Preferences page.
```

Reply:
fixed